ui/client/store/configureStore.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable i18next/no-literal-string */
```

Comment:
In this change you only rewritten this class to TS or it was changed anyway?

Reply:
like in commit message, added state persistency